salt/modules/file.py

```diff
@@ -344,7 +344,7 @@ def user_to_uid(user):
     try:
         if isinstance(user, int):
             return user
-        return pwd.getpwnam(user).pw_uid
+        return pwd.getpwnam(user.encode('utf-8')).pw_uid
```

Comment:
You should use `salt.utils.stringutils` instead.

Reply:
Hi Bo.
Fixed in 384a726

Reply:
Yep, now it is the right thing. Thanks!